doc/website/concepts/best-practice-for-testing.md

```diff
@@ -55,7 +55,7 @@ The catchwords can be used to draw simple scenarios which nicely fits to the AAA
   - etc.
 - etc.
 
-Following [Hyrum's Law](https://www.hyrumslaw.com/) loosely, given enough users, one will find ways to use the software in a way it was never imagined.
+Following [Hyrum's Law](https://web.archive.org/web/20220728001604/https://www.hyrumslaw.com) loosely, given enough users, one will find ways to use the software in a way it was never imagined.
```

Comment:
It seems that the website changed something in their security settings so that HTTPS is not enabled, I guess the linkchecker expect to have it enabled (which is good).

The reason for using the wayback machine is to have static version of the website to the corresponding link. The linkchecker will never detect if someone modifies the content of a website so that the context of the link here may not fit to the actual content of the website.

Since this problem occurs here again when reverting the change, I'm open for solutions to solve this.

Reply:
@dkroenke This was a temporary issue. Everything is back to normal now. I understand what you mean. However, that would mean we would need to use static wayback machine links everywhere. That's overkill for me. Could you please change the link back to the normal one?

Reply:
Sorry but there is nothing back to normal, the website's certificate expired on Saturday, July 30 which makes it impossible to use HTTPS at this point. You can see this by the red warning near to the web site address in your browser (it should indicate that somehow).
This is why the linkchecker refuses to check the website. Changing the link to use `http` only does not solve this.
Changing one link here does not mean that every link shall be changed.